src/controllers/authController.js

```diff
@@ -101,3 +101,42 @@ exports.resetPassword = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.login = async (req, res) => {
+	try {
+		const prevUser = await User.findOne({ email: req.body.email});
+		console.log(prevUser)
+		if (prevUser) {
+			const passMatch= await bcrypt.compare(
+				req.body.password,
+				prevUser.password
+			);
+
+			if (passMatch && req.body.email === prevUser.email) {
```

Comment:
checking `req.body.email === prevUser.email` is not necessary because prevUser only exists because the email is `req.body.email`